src/main.rs

```diff
@@ -21,9 +21,15 @@ mod error;
 mod workspace;
 
 /// Structure of `package.json`.
-#[derive(Debug, Clone, Deserialize, Serialize)]
+#[derive(Debug, Clone, Deserialize, Serialize, PartialEq)]
```

Comment:
You don't need `PartialEq` if you convert it to `serde_json::Value`. See https://github.com/pnpm/pn/pull/39/files/2c455a0a3fe8c40158eb78dc7988efce64913461#r1233188905

Reply:
comparing `serde_json::Value` also fails. When converting NodeManifest to Value, the Value does include the empty name and version property. On the other while, creating a Value from a json file without name,version excludes it. So the two results are not the same.
![image](https://github.com/pnpm/pn/assets/91675932/5f69de8c-5f0d-4c6c-9e4b-2f456a69a8fc)
^ From converting `{"scripts": {"test": "echo hello world"}}` to Value, the results end up being different.

I think asserting NodeManifest is fine? Since the read_package_manifest function also converts the output to NodeManifest we can make sure both reading the json and converting to NodeManifest works. Comparing Values wouldn't check for comparing NodeManifest. But if u still want to compare values, i can try comparing on the Script's part?
For example
```rs
assert_eq!(received.get("scripts"), expected.get("scripts"));
```
Ignoring version and name.

Reply:
Originally, I expected you to also add `"name": "", "version": ""` as well. But now that I think about it, this approach is not scalable in the long term (adding new fields to `NodeManifest` would force programmer to add these fields to every test ever written). So yeah, I guess comparing `NodeManifest` is the only way now.